apex/contrib/groupbn/batch_norm.py

```diff
@@ -4,6 +4,16 @@
 
 import bnp
 
+def check_if_rocm_pytorch():
+    is_rocm_pytorch = False
+    if torch.__version__ >= '1.5':
+        from torch.utils.cpp_extension import ROCM_HOME
+        is_rocm_pytorch = True if ((torch.version.hip is not None) and (ROCM_HOME is not None)) else False
+
+    return is_rocm_pytorch
+
+IS_ROCM_PYTORCH = check_if_rocm_pytorch()
```

Comment:
This a nitpick but could we unifiy or import this from the setup.py definition.

Reply:
I will look into this

Reply:
I have not refactored this code.  I think it will need a file restructuring in APEX.  So, I decided to leave this to the APEX team